### ISSUE TITLE
OOM when allocating tensor with shape[2259000,300]

### ISSUE THREAD

**Issue by kepsail** [StephanieWyt/RDGCN#3]
Caused by op 'gradients/concat', defined at:
  File "main.py", line 34, in <module>
    Config.epochs, train, e, Config.k, test)
  File ".\include\Model.py", line 275, in training
    train_step = tf.train.AdamOptimizer(learning_rate).minimize(loss)
